platform/iphone/app_delegate.h

```diff
@@ -28,20 +28,14 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 
-#import "gl_view.h"
-#import "view_controller.h"
+#import <Foundation/Foundation.h>
 #import <UIKit/UIKit.h>
 
-#import <CoreMotion/CoreMotion.h>
+#import "gl_view.h"
 
-@interface AppDelegate : NSObject <UIApplicationDelegate, GLViewDelegate> {
-	//@property (strong, nonatomic) UIWindow *window;
-	ViewController *view_controller;
```

Comment:
This pattern is particularly ancient style now that llvm synthesizes all `@property` and backs them with appropriate instance variables that we can name and use without `@dynamic` or `@synthesize` workarounds.